veracruz-server/src/veracruz_server_nitro.rs

```diff
@@ -11,24 +11,14 @@
 
 #[cfg(feature = "nitro")]
 pub mod veracruz_server_nitro {
-    use crate::ec2_instance::EC2Instance;
     use crate::veracruz_server::{VeracruzServer, VeracruzServerError};
-    use lazy_static::lazy_static;
-    use std::sync::Mutex;
-    use veracruz_utils::platform::Platform;
-    use veracruz_utils::platform::nitro::nitro::{NitroRootEnclaveMessage, RuntimeManagerMessage, NitroStatus};
-    use veracruz_utils::platform::nitro::nitro_enclave::{NitroEnclave, NitroError};
+    use std::io::Read;
+    use veracruz_utils::platform::nitro::nitro::{RuntimeManagerMessage, NitroStatus};
```

Comment:
Can these related imports be grouped?

Reply:
Which ones are you asking about? The one above this is already grouped.